docker-compose.yml

```diff
@@ -1,7 +1,34 @@
-# docker-compose.ml
-dc-campaign-ui:
-  build: .
-  ports:
-  - "3001:3001"
-  environment:
-  - REACT_WEBPACK_ENV=dev
+version: '2'
+services:
+  dc-campaign-finance-ui:
+    build: .
+    ports:
+    - "3001:3001"
+    environment:
+    - REACT_WEBPACK_ENV=dev
+  redis:
+    image: redis
+    ports:
+    - "6379"
+    command: [redis-server]
+  mongodb:
+    image: mongo:3.2
+    ports:
+    - "27017:27017"
+  mongo-seed:
+    build:
+      context: .
+      dockerfile: Dockerfile-mongo-seed
+    links:
+     - mongodb
+  dc-campaign-finance-api:
```

Comment:
This is maybe better as a next step, but it would be awesome if we could also run this service on EC2, in order to do so we would also need nginx image that forwards 80.